### DISCUSSION TITLE
How do I make catchall actually reset the submap?

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5020] [Q&A] [answered]
```
#submaps
bind = ALT, a, submap,onehand
submap=onehand
bind = , e, movefocus, r
bind = , a, movefocus, l
bind = ,ESCAPE,submap,reset
bind = ,catchall,submap,reset
submap=reset

```

I have this in my config file, according to the docs, having that catchall at the bottom should make it so that any unknown key that is pressed sends me back to my default submap

Any idea why that isn't happening? am I misunderstanding how it works?

I really don't want to have to make every key that isn't bound switch to a submap manually, that sounds like a lot of work.

**Comment by vaxerski** [comment, marked as answer]
maybe outdated?

**Comment by Kommynct** [comment]
Yup, didn't realize the most recent release was so recent and not in the arch repos yet
